src/index.js

```diff
@@ -99,26 +99,24 @@ export class WT {
     return `//${this.context.location.hostname}`;
   }
 
-  sendToServer(payload) {
+  sendToServer(payload, resolve, reject) {
     this.loaderImage = this.loaderImage || this.getLoaderImage();
     const query = QS.stringify(
       payload,
       { addQueryPrefix: false, ...(this.wtConfig.stringifyOptions || DEFAULT_STRINGIFY_OPTIONS) },
     );
 
-    return new Promise((resolve, reject) => {
```

Comment:
question -- instead of using a standard polyfill, can we not do it at the transpiler level? something like https://www.npmjs.com/package/babel-plugin-es6-promise

Reply:
@bcharna I think we may want to take on a strategy like that if we use Promises more heavily. I'm not clear if using a polyfill like that introduces bloat (since it then needs to load a Promise library to package with the wt client).

Reply:
okay sounds good. would be good to look into if we use them more